db/update.sql

```diff
@@ -1,6 +1,6 @@
 -- drivers license information
 CREATE TABLE IF NOT EXISTS `UserDriverLicenses` (
-  `user_id` int(11) NOT NULL,
+  `user_id` int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
Why do you make this field "AUTO_INCREMENT"?

Note below:

```
ADD CONSTRAINT `userdriverlicenses_ibfk_1` FOREIGN KEY (`user_id`) REFERENCES `User` (`UID`) ON DELETE CASCADE ON UPDATE CASCADE;
```
this field is a **foreign key**